qa/tasks/cephfs/test_volumes.py

```diff
@@ -832,6 +834,8 @@ def test_clone_subvolume_info(self):
                 raise RuntimeError("%s not present in the metadata of subvolume" % md)
         if subvol_info["type"] != "clone":
             raise RuntimeError("type should be set to clone")
+        if subvol_info["pool_namespace"] == "default":
```

Comment:
can you instead check the value of 'pool_namespace' since we know what to expect?

Reply:
Done